HackingWithSwift/week1/day_5/Accepting parameters/Accepting parameters.playground/Contents.swift

```diff
@@ -0,0 +1,13 @@
+func example (number: Int) {
+    print(number*number/2)
```

Comment:
It would be clearer if we added spaces around the operators, which is a common convention in Swift to enhance code readability. I suggest updating the line to:
`print(number * number / 2)`